src/types.ts

```diff
@@ -79,6 +79,7 @@ export type ObjectsStat = {
     count: number;
     size: number;
     unpackedSize: number;
+    unpackedRestoredSize: number;
```

Comment:
@lahmatiy do we want to have more narrow types for these stat objects, or I should leave it as is?